pkgs/development/tools/vend/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
```

Comment:
I'm kind of surprised that this builds without `vendorSha256 = null` being set?

Reply:
Default is null for vendorSha256 :) So if your package is fully vendored, you don't need to do anything, just set src and it will build :D

Reply:
Why are we setting `null` in packages then?

Reply:
Opened a PR: https://github.com/NixOS/nixpkgs/pull/89454